clients/csharp-wpf/VoiceAssistantClient/VoiceAssistantClient.csproj

```diff
@@ -43,6 +43,7 @@
     <BootstrapperEnabled>true</BootstrapperEnabled>
   </PropertyGroup>
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'Debug|x64'">
+    <RuntimeIdentifier>win-x64</RuntimeIdentifier>

```

Comment:
What are these identifier's? Why didn't we have them before? Would a new project have to add these?

Reply:
All good questions. Since this fixed the problem for me and not for Fan, I'm abandoning this PR.